### ISSUE TITLE
Hellbender Driver Turret

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#117]
- ANIMATION RELATED
  - [ ] UT2004 Hellbender is missing animations on both turrets when getting in and out of the vehicle, in UT3 they kind of raise or wake up by the look of it as the Hellbender raises up

 - GAME PLAY RELATED
  - [ ] Driver Turret doesn't turn instantly in UT3 but does turn faster

**Comment by HellDragon-HK** [comment]
@GreatEmerald Found that the Skymine combo has some different values but I don't know where to fix it, I assume it's the ````ComboDamageType=class'DamTypePRVLaser'````  but that's not DamTypePRVCombo  so where is that at?

It needs 
    KDamageImpulse=2000
    VehicleMomentumScaling=1.5
    VehicleDamageScaling=+1.0  (it has less than the default so this should override it and return to default which is what UT3 has I think)

**Comment by GreatEmerald** [comment]
It's this line: http://greatemerald.xmpcommunity.com/uncodex/UT2004/Source_onslaught/onsskymine.html#63
`ComboDamageType` is actually the DamageType that gets applied to sky mines when the beam bounces from one to the other.

**Comment by HellDragon-HK** [comment]
I know I marked it out but with the new corona changes the turret's missing corona is very noticeable now, so @GreatEmerald is there a way to get a corona on a turret?

**Comment by GreatEmerald** [comment]
Certainly, just port over this code block: http://greatemerald.xmpcommunity.com/uncodex/UT2004/Source_onslaught/onsvehicle.html#493 (and all the associated variables).

**Comment by HellDragon-HK** [comment]
Haven't had much luck with PostNetPlay, can you be more specific please on what does and doesn't go?

**Comment by GreatEmerald** [comment]
Put up a pull request, then.

**Comment by HellDragon-HK** [comment]
I haven't actually tried it yet, just thinking on how the Manta went.....ok trying it and I get this
C:\Unreal Anthology\UT2004\UT3Vehicles\Classes\UT3HellbenderSideGun.uc(53) : Error, Bad or missing expression after '<'

**Comment by GreatEmerald** [comment]
Yea, that doesn't tell me anything. Put up a pull request.
